src/surabhi/assignment31/SecondMaxNumber.java

```diff
@@ -0,0 +1,29 @@
+package surabhi.assignment31;
+
+/*WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 */
+
+public class SecondMaxNumber {
+	void getSecMaxNumber(int[] arr) {
+		int max=arr[0];int count=0;
+		for(int index=1;index<arr.length;index++) {
+			if(max<arr[index]) {
+				max=arr[index];
+				count++;
```

Comment:
this is will fails for below inout
int input[] = {10, 33, 43, 55, 97, 111, 99, 3};


to handle all the scenario
 for (int index = 0; index < input.length; index++) {

            if (max < input[index]) {
                secondMax = max;
                max = input[index];
            } else if (input[index] > secondMax && max != input[index]) {
                secondMax = input[index];
            }
        }